libs/core/datastructures/tests/performance/small_vector_benchmark.cpp

```diff
@@ -19,61 +20,85 @@
 
 ///////////////////////////////////////////////////////////////////////////////
 template <typename Container>
-std::uint64_t measure(std::size_t repeat, std::size_t size)
+void fill(std::size_t size)
 {
-    std::uint64_t start = hpx::chrono::high_resolution_clock::now();
-    for (std::size_t i = 0; i != repeat; ++i)
+    Container cont;
+    for (std::size_t i = 0; i != size; ++i)
     {
-        Container cont;
-        for (std::size_t i = 0; i != size; ++i)
-        {
-            cont.push_back(typename Container::value_type{});
-        }
+        cont.push_back(typename Container::value_type{});
     }
-    return (hpx::chrono::high_resolution_clock::now() - start) / repeat;
 }
 
-template <typename T, std::size_t N>
-void compare(std::size_t repeat, std::size_t size)
-{
-    std::uint64_t time = measure<hpx::detail::small_vector<T, N>>(repeat, size);
+// template <typename T, std::size_t N>
+// void compare(std::size_t repeat, std::size_t size)
+// {
+//     std::uint64_t time = measure<hpx::detail::small_vector<T, N>>(repeat, size);
 
-    std::cout << "-----Average-(hpx::small_vector<" << typeid(T).name() << ", "
-              << N << ">)------ \n"
-              << std::left << "Average execution time : " << std::right
-              << std::setw(8) << time / 1e9 << "\n";
-}
+//     std::cout << "-----Average-(hpx::small_vector<" << typeid(T).name() << ", "
+//               << N << ">)------ \n"
+//               << std::left << "Average execution time : " << std::right
+//               << std::setw(8) << time / 1e9 << "\n";
+// }
```

Comment:
Can be removed

Reply:
Removed in https://github.com/STEllAR-GROUP/hpx/pull/6448/commits/c35e4e4f3ca2332c6f08323c9e3504175bede47d